core/routemgmt/createRoute.js

```diff
@@ -0,0 +1,98 @@
+/**
```

Comment:
yes, it's a copy from the cloudant package.  it's really just a POC placeholder to get a good feel for cloudant/couch operations.

The cloudant package is a Bluemix only feature and this API GW feature is targeted for openwhisk (last I understood), the cloudant package could not be a dependency.